### ISSUE TITLE
Aortic annulus calcification

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6575]
**Preferred term label:**  Aortic annulus calcification

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
Aortic valve calcification HP:0004380

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added with definition

Pathological deposition of calcium salts in the aortic annulus, a fibrous ring-like structure found at the insertion point of the basal attachments of the aortic valve leaflets within the left ventricular outflow tract.
